app/assets/v2/js/pages/bounty_details.js

```diff
@@ -826,7 +865,8 @@ var do_actions = function(result) {
   const current_user_is_approved = is_current_user_approved(result);
   // which actions should we show?
   const should_block_from_starting_work = !is_interested && result['project_type'] == 'traditional' && (result['status'] == 'started' || result['status'] == 'submitted');
-  let show_start_stop_work = is_still_on_happy_path && !should_block_from_starting_work && is_open && !isBountyOwner(result);
+  let show_start_stop_work = is_still_on_happy_path && !should_block_from_starting_work &&
```

Comment:
@thelostone-mc I moved to the `callbacks` object in bounty_details.js . 
The start_stop button is now hidden if the issue is reserved for another.